gslab_scons/builders/build_latex.py

```diff
@@ -36,6 +36,6 @@ def add_call_args(self):
         '''
         '''
         target_name = os.path.splitext(self.target[0])[0]
-        args = '%s %s %s > %s' % (self.cl_arg, target_name, self.source_file, self.log_file)
+        args = '%s %s %s > %s' % (self.cl_arg, target_name, os.path.normpath(self.source_file), os.path.normpath(self.log_file))
```

Comment:
Do we need to norm the `target_name`?

Reply:
That's just a "name" so I think we're fine?